### PR TITLE
Double origin plane S66 does not have a cut point P204

### PR DESCRIPTION
**Pull request by StevenClontz**
Answers https://topology.pi-base.org/spaces/S000066/properties/P000204

Note the change in name for S66.

### CODE REVIEW

**Review by GeoffreySangston — APPROVED** [review]

**Comment by StevenClontz** [comment]
> Do you think the user gilgamesh confused this space for a two-dimensional analog of [Line with two origins](https://topology.pi-base.org/spaces/S000083)?

Absolutely I did. Actually, I think a 1D analog of this and a 2D analog of S83 would both be interesting examples, though they'd need justification with some interesting combination of properites not already present on pi-Base before we added them. I like adding the warning, so I'll slip that in before merging.

**Comment by StevenClontz** [comment]
@GeoffreySangston check 6a6a8aa and re-approve+merge if appropriate.

**Review by GeoffreySangston — APPROVED** [review]
I like it!

File: spaces/S000066/properties/P000204.md
Comment (prabau) [inline spaces/S000066/properties/P000204.md:11]: That seem unnecessarily complicated.  Why not say the space remains connected after removing any single point?
Actually, it's a triviality.

**Pull request merged**
